showcase/tests/helpers/index.ts

```diff
@@ -7,10 +7,9 @@ import {
   setupApplicationTest as upstreamSetupApplicationTest,
   setupRenderingTest as upstreamSetupRenderingTest,
   setupTest as upstreamSetupTest,
+  type SetupTestOptions,
 } from 'ember-qunit';
 
-import type { SetupTestOptions } from 'ember-qunit';
```

Comment:
Eve thought ember doesn't complain about this we should keep type imports in separate line for consistency and in future it might start complaining about this type of imports.